architecture/modules/understanding-cluster-version-operator.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-authentication.adoc
+[id="understanding-cluster-version-operator_{context}"]
+= Understanding the Cluster Version Operator (CVO)
```

Comment:
This needs to be combined with the existing /modules/operators-overview.adoc file.